pglast/printers/ddl.py

```diff
@@ -3363,16 +3363,16 @@ def variable_set_stmt(node, output):
             output.print_list(node.args, ',')
         else:
             output.print_name(node.name.split('.'))
+        if node.kind == vsk.VAR_SET_CURRENT:
+            output.write('FROM CURRENT')
+        else:
             output.write(' TO ')
         if node.kind == vsk.VAR_SET_VALUE:
             output.print_list(node.args)
         elif node.kind == vsk.VAR_SET_DEFAULT:
             output.write('DEFAULT')
         elif node.kind == vsk.VAR_SET_MULTI:
             pass
-        else:
-            raise NotImplementedError("SET statement of kind %s not implemented yet"
-                                      % node.kind)
```

Comment:
`VAR_SET_CURRENT` is the last option, so it doesn't make sense leaving this here anymore.